product-service/serverless.yml

```diff
@@ -16,6 +16,13 @@ provider:
   region: eu-west-1
   profile: personal
 
+  environment:
+    PG_HOST: lesson4-instance.cei7owdrptoz.eu-west-1.rds.amazonaws.com
+    PG_PORT: 5432
+    PG_DATABASE: postgres
+    PG_USERNAME: postgres
+    PG_PASSWORD: # password
```

Comment:
What about password? Or you set it manually every time?

Reply:
Task didn`t have any restrictions against not using .env, I just didn`t push password filed to github, that`s not against rules.